### PR TITLE
Improve index page

### PR DESCRIPTION
**Pull request by zzet**

### CODE REVIEW

**Comment by zzet** [comment]
@jbmeerkat What do you mean about this variant?

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class WelcomeController < ApplicationController
+  layout 'lending'
```
Comment (jbmeerkat) [inline app/controllers/welcome_controller.rb]: @zzet `layout`

Comment (jbmeerkat) [inline app/controllers/welcome_controller.rb]: @zzet `landing`

**Comment by jbmeerkat** [comment]
:+1:

**Pull request merged**